### PR TITLE
update install guides

### PR DESCRIPTION
**Pull request by samos123**
Merge once a new release is out that includes updated gcp managed infra piece

### CODE REVIEW

File: docs/installation/gcp.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
+---
+sidebar_position: 4
+---
+
+# GCP Install Guide
+
+
+## Automatic Install
```
Comment (nstogner) [inline docs/installation/gcp.md:8]: I would avoid splitting into an "automatic" and "advanced" section. I don't see a reason to use docker now that we are moving away from terraform. The primary tool that the user will need is `gcloud` and they will need that installed regardless of whether they use docker or not.

I think each install page could just:

1. list any required quotas / etc for the cloud account
2. list bulletpoints of all things that are installed (cluster, bucket, registry)
3. embed the install script directly into the markdown to show the steps

**Review by nstogner — COMMENTED** [review]
PS: Missing `_category.json` file in `installation/` directory.

**Comment by samos123** [comment]
> PS: Missing `_category.json` file in `installation/` directory.

it seems to automatically use the info from installation/installation.md

File: docs/installation/installation.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+gets installed.
+
+## Automated install
+The basic automatic installer creates a new K8s cluster, new bucket, new principal and automatically
+sets up everything that's needed for Substratus to work.
+
+## Customizable install
+The advanced install lets you use your existing K8s cluster and other resources instead
+of having Substratus automatically do everything for you. Users in restricted environments
+or who need to re-use existing resources are encouraged to use the advanced install.
```
Comment (nstogner) [inline docs/installation/installation.md:25]: There is some mixed terminology here: `Automated install`, `basic automatic installer`, ` Customizable install`, `advanced install`. I would standardize on 2 names: `Basic` and `Customized`.

File: docs/installation/gcp.md [hunk 3]
```diff
@@ -0,0 +1,227 @@
+---
+sidebar_position: 4
+---
+
+# GCP Install Guide
+
+## Subsratus required resources
+* GKE Cluster with GCSfuse and Workload Identity
```
Comment (nstogner) [inline docs/installation/gcp.md:8]: `GCS FUSE`

File: docs/installation/gcp.md [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+* GCS Bucket to store Model and Dataset data
+* Google Artifact Registry Container Repository to store newly built container images
+* Google Service Account for accessin GCS Bucket, Artifact Registry and managing cross
```
Comment (nstogner) [inline docs/installation/gcp.md:11]: `accessin` --> `read/write access to`

Comment (nstogner) [reply docs/installation/gcp.md:11]: I would use sub-bullets for the access that this SA will need:

* Read/Write: Objects within Google Cloud Storage Bucket
* Read/Write: Artifacts within Google Artifact Registry
* Read/Write: Workload Identity bindings

Comment (samos123) [reply docs/installation/gcp.md:11]: done

File: docs/installation/installation.md (hunk 2 above)
Comment (samos123) [reply docs/installation/installation.md:25]: done

**Review by nstogner — APPROVED** [review]

**Pull request merged**